Gruntfile.js

```diff
@@ -14,7 +14,9 @@ const extract = require('jsxgettext-recursive');
 // use this if you want to recursively match all subfolders:
 // 'test/spec/**/*.js'
 module.exports = function (grunt) {
-  'use strict';
+  'use strict'
```

Comment:
Yeah that could work. Now we need a rule to throw errors if there *are* semicolons found. :sweat_smile: